src/pages/settings/Profile/PersonalDetails/AddressPage.js

```diff
@@ -52,175 +52,158 @@ const defaultProps = {
     },
 };
 
-class AddressPage extends Component {
-    constructor(props) {
-        super(props);
+/**
+ * Submit form to update user's first and last legal name
+ * @param {Object} values - form input values
+ */
+function updateAddress(values) {
+    PersonalDetails.updateAddress(values.addressLine1.trim(), values.addressLine2.trim(), values.city.trim(), values.state.trim(), values.zipPostCode.trim(), values.country);
+}
 
-        this.validate = this.validate.bind(this);
-        this.updateAddress = this.updateAddress.bind(this);
-        this.onCountryUpdate = this.onCountryUpdate.bind(this);
+function AddressPage(props) {
+    const {translate} = props;
+    const [countryISO, setCountryISO] = useState(PersonalDetails.getCountryISO(lodashGet(props.privatePersonalDetails, 'address.country')) || CONST.COUNTRY.US);
+    const isUSAForm = countryISO === CONST.COUNTRY.US;
 
-        const currentCountry = lodashGet(this.props.privatePersonalDetails, 'address.country') || '';
-        const currentCountryISO = PersonalDetails.getCountryISO(currentCountry) || CONST.COUNTRY.US;
-        const zipSampleFormat = lodashGet(CONST.COUNTRY_ZIP_REGEX_DATA, [currentCountryISO, 'samples'], '');
-        this.state = {
-            isUsaForm: currentCountry === CONST.COUNTRY.US || currentCountry === CONST.USA_COUNTRY_NAME,
```

Comment:
Even if `United States` comes from the back end, in this refactor we initialize `countryISO ` as `PersonalDetails.getCountryISO(lodashGet(props.privatePersonalDetails, 'address.country')) || CONST.COUNTRY.US` before comparing, so it would have been transformed to `US` anyway.

Reply:
Removed